roles/postgres/tasks/main.yml

```diff
@@ -42,28 +42,17 @@
     enabled: true
   become: true
 
-- name: Setup postgresql databases
-  community.postgresql.postgresql_db:
-    name: "{{ item.database }}"
-  with_items: "{{ postgres_users }}"
-  become: true
-  become_user: postgres
-
 - name: Setup postgresql users
   community.postgresql.postgresql_user:
-    db: "{{ item.database }}"
     user: "{{ item.username }}"
   with_items: "{{ postgres_users }}"
   become: true
   become_user: postgres
 
-- name: Grant users permissions to create tables in the schema `public`
```

Comment:
I now think that my fix in 3ccb6526 was wrong an I misunderstood what changed in PostgreSQL 15. The database owner will have the necessary permissions on the schema `public`, and no additional grants are needed (https://www.enterprisedb.com/blog/new-public-schema-permissions-postgresql-15).